### DISCUSSION TITLE
Copilot X Access

### DISCUSSION THREAD

**Discussion by ahmedivy** [community/community#56597] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I got access to Copilot X a few day ago. It wants me to download VS Code Insiders but why?

**Comment by davecheney** [comment, marked as answer]
@ahmedivy currently the chat integration requires features which are only available in the -insiders build. In time this restriction will be relaxed when those features are promoted to the mainline vscode build.

**Reply by ahmedivy** [reply]
How much time will it take approximately?

**Reply by davecheney** [reply]
I'm sorry I don't know. I would expect a month or two, but that's just a guess, please don't hold me too it.

**Reply by ahmedivy** [reply]
Ok. Thank you🤗

**Reply by luixo** [reply]
@davecheney 
Do you know if that's a policy or there are any technical limitations?
If the latter - I would probably track the feature / PR / merge progress.

**Comment by pankajb23** [comment]
Please we need co-pilot x full features for intellij
